tests/local/mod_accredible_accredible_test.php

```diff
@@ -47,7 +47,7 @@ public function test_save_record() {
         global $DB;
 
         // When creating a new record.
-        $post = $this->generatePostObject();
+        $post = $this->generatepostobject();
```

Comment:
For consistency even though function names are case-insensitive.

c.f. https://www.php.net/manual/en/functions.user-defined.php

> Note: Function names are case-insensitive for the ASCII characters A to Z, though it is usually good form to call functions as they appear in their declaration.